docs/publishers/cli.md

```diff
@@ -80,42 +97,57 @@ To publish a Data Package, go to the Data Package directory (with `datapackage.j
 run:
 
 ```bash
-dpm publish
+data push
 ```
 
 If your configured *username* and *access_token* are correct, dpm will
 upload datapackage.json and all relevant resources to the registry server.
 
-### Tag
-
-To create historical snapshot of your data, you can tag previously uploaded datapackage on the server. Use `dpm tag` command:
+### Download
 
+To download run the following command:
 ```bash
-cd datapackage-dir
-dpm tag v1.1
+data get <publisher>/<package>
 ```
-This will copy the latest version of the Data Package to a separate location in the BitStore. This way you will be able keep a copy of your Data Package at this particular point in time.
+New Data Package will be downloaded into current working directory. 
```

Comment:
New Data Package will be downloaded into the current working directory replicating structure of a Data Package, e.g., `owner/name/datapackage.json`.